app/viewmodels/checkAnswers/returns/ImportedPlasticPackagingWeightSummary.scala

```diff
@@ -26,13 +26,17 @@ import viewmodels.checkAnswers.SummaryViewModel
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
 
-class ImportedPlasticPackagingWeightSummary private (key: String) extends SummaryViewModel {
+class ImportedPlasticPackagingWeightSummary (key: String) extends SummaryViewModel {
```

Comment:
we want to keep these as private and use the static objects in the companion. We really want to do these for the others too, but fair if we wanna get these in and do in a seperate PR so this can be demo'd at show&tell